ddtrace/debugging/_debugger.py

```diff
@@ -251,14 +251,20 @@ def __init__(self, tracer=None):
             raise_on_exceed=False,
         )
 
+        if config.exception_debugging:
+            from ddtrace.debugging._exception.auto_instrument import SpanExceptionProcessor
+
+            SpanExceptionProcessor(collector=self._collector).register()
+
         # TODO: this is only temporary and will be reverted once the DD_REMOTE_CONFIGURATION_ENABLED variable
         #  has been removed
         if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", True)) is False:
```

Comment:
should we move this under if config.enabled at line 266? no need to change this if not enabled?

Reply:
ah yes, good catch!